src/views/home.hbs

```diff
@@ -15,4 +21,10 @@
     </a>
     {{/each}}
 </div>
-<a href="/painting-form"><button>Add Painting</button></a>
\ No newline at end of file
+<a href="/painting-form"><button>Add Painting</button></a>
+<script>
+    document.getElementById("searchImg").addEventListener("click", e => {
```

Comment:
Handlebars replaces the DOM, you can't do this in the backend as there's no document.